docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -6,14 +6,14 @@ dev_langs:
   - "vb"
 author: thraka
 ms.author: adegeo
-ms.date: 05/06/2019
+ms.date: 06/14/2019
 ---
 
-# What's new in .NET Core 3.0 (Preview 5)
+# What's new in .NET Core 3.0 (Preview 6)
 
-This article describes what is new in .NET Core 3.0 (through preview 5). One of the biggest enhancements is support for Windows desktop applications (Windows only). By using the .NET Core 3.0 SDK component Windows Desktop, you can port your Windows Forms and Windows Presentation Foundation (WPF) applications. To be clear, the Windows Desktop component is only supported and included on Windows. For more information, see the [Windows desktop](#windows-desktop) section later in this article.
+This article describes what is new in .NET Core 3.0 (through preview 6). One of the biggest enhancements is support for Windows desktop applications (Windows only). By using the .NET Core 3.0 SDK component Windows Desktop, you can port your Windows Forms and Windows Presentation Foundation (WPF) applications. To be clear, the Windows Desktop component is only supported and included on Windows. For more information, see the [Windows desktop](#windows-desktop) section later in this article.
 
-.NET Core 3.0 adds support for C# 8.0. It's highly recommended that you use the latest release of Visual Studio 2019 Update 1 Preview or VSCode with the OmniSharp extension.
+.NET Core 3.0 adds support for C# 8.0. It's highly recommended that you use the latest release of Visual Studio 2019 or Visual Studio Code with the OmniSharp extension.
```

Comment:
I think it's strongly recommended to use 16.1 preview for .NET Core 3.0 (or C# 8).  @Thraka would know for sure.

Reply:
What's confusing me is that if I go here:
https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes

The current version is already 16.1. And 16.2 is currently in [Preview](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview).

So the exact version needs to be clear here. Update 1 Preview doesn't exist.

@Thraka please confirm what's needed.

Reply:
[The download page](https://dotnet.microsoft.com/download/dotnet-core/3.0) says .NET Core 3.0 should be used with the latest previews of Visual Studio (Visual Studio 2019/Visual Studio for Mac 8.2).

Reply:
If you are using Visual Studio 2019 (non Preview), you need to enable the option manually.

![image](https://user-images.githubusercontent.com/45985406/59870145-83c4d280-93c7-11e9-8215-29e98ea6c6d5.png)

Reply:
Now that I think about it more I think it should say latest Preview. As VS is continually iterating with .NET Core, anything that's released in a Core preview is best supported by a VS preview.

Reply:
I like your proposal @Thraka.